CMakeLists.txt

```diff
@@ -14,6 +14,11 @@ if(DEFINED ENV{VCPKG_ROOT})
     message(STATUS "vcpkg found VCPKG_ROOT=$ENV{VCPKG_ROOT}")
     # https://github.com/microsoft/vcpkg/blob/master/docs/users/integration.md#changing-the-triplet-1
 
+    if(DEFINED ENV{VECTORCXX_VCPKG_MANIFEST_DIR})
```

Comment:
Moved vcpkg manifest files from root path to ci/dev_image/, so we need this env to find correct manifest path

Reply:
What if we don't move the manifest file? Since the CMakeLists.txt is generated, we should avoid changing it manually.

Reply:
We have a hook `include(${PROJECT_CMAKE_DIR}/project.cmake)`, do you think if we could put this into `project.cmake` if needed?